src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Components/TagHelperDescriptorExtensions.cs

```diff
@@ -46,6 +48,53 @@ public static bool IsGenericTypedComponent(this TagHelperDescriptor tagHelper)
             string.Equals(bool.TrueString, value);
     }
 
+    /// <summary>
+    /// Given a taghelper binding it finds the BoundAttribute that is a type parameter and then the
+    /// actual binding value for that type.
+    ///
+    /// <code>
+    /// &lt;MyTagHelper
+    ///   TItem="string"
```

Comment:
If there are multiple generic arguments, this method will return just the first one, right? Is that okay (and should it be mentioned in the doc comment) or would it be better if the method returned `false` in that case?

Reply:
Correct. I'll change it so we return false in that case and add a remark in the doc comment